package.json

```diff
@@ -48,6 +51,7 @@
     "pg-hstore": "^2.3.2",
     "request": "^2.87.0",
     "sequelize": "^4.38.0",
+    "should": "^13.2.3",
```

Comment:
I understand why you felt you needed to install this package, but we're already using Chai, which has a `should` function. That makes this package redundant.